src/syntax/rules.rkt

```diff
@@ -44,24 +69,6 @@
         [(define (write-proc rule port mode)
            (fprintf port "#<rule ~a>" (rule-name rule)))])
 
-(define (rule-ops-supported? rule)
-  (define (ops-in-expr expr)
-    (cond
-      [(list? expr)
-       (and (impl-exists? (car expr))
-            (for/and ([subexpr (cdr expr)])
-              (ops-in-expr subexpr)))]
-      [else true]))
-  (ops-in-expr (rule-output rule)))
-
-(register-reset
- #:priority 10 ; Must be higher than priority for pruning operators
- (λ ()
-   (*rulesets*
-    (for/list ([ruleset (*rulesets*)])
-      (match-define (list rules groups types) ruleset)
-      (list (filter rule-ops-supported? rules) groups types)))))
-
```

Comment:
I think we might potentially no longer need `#:priority` on resetters, which is also nice.